client/src/index.jsx

```diff
@@ -2,10 +2,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import About from './components/about';
+import Pledges from './components/pledges';
 
 const App = () => (
   <div>
+    <div>
     <About />
```

Comment:
I think your About component should be indented. Also consider dropping the parent divs of the About and Pledges components altogether unless ESLint tells you to keep them.